nmigen_boards/nexys4ddr.py

```diff
@@ -0,0 +1,172 @@
+    device      = "xc7a100t"
+    package     = "csg324"
+    speed       = "1"
+    default_clk = "clk100"
+    resources   = [
+        Resource("clk100", 0, Pins("E3", dir="i"),
+                 Clock(100e6), Attrs(IOSTANDARD="LVCMOS33")),
+
+        *SwitchResources(pins="J15 L16 M13 R15 R17 T18 U18 R13 "
+                              "T8  U8  R16 T13 H6  U12 U11 V10",
+                         attrs=Attrs(IOSTANDARD="LVCMOS33")),
+
+        *LEDResources(pins="H17 K15 J13 N14 R18 V17 U17 U16 "
+                           "V16 T15 U14 T16 V15 V14 V12 V11",
+                      attrs=Attrs(IOSTANDARD="LVCMOS33")),
+
+        RGBLEDResource(0, r="N15", g="M16", b="R12",
+                       attrs=Attrs(IOSTANDARD="LVCMOS33")),
+        RGBLEDResource(1, r="N16", g="R11", b="G14",
+                       attrs=Attrs(IOSTANDARD="LVCMOS33")),
+
+        Display7SegResource(0,
+                            a="T10", b="R10", c="K16", d="K13", e="P15",
+                            f="T11", g="L18", dp="H15", invert=True,
+                            attrs=Attrs(iIOSTANDARD="LVCMOS33")),
+        Resource("display_7seg_an", 0,
+                 PinsN("J17 J18 T9 J14 P14 T14 K2 U13", dir="o"),
+                 Attrs(iIOSTANDARD="LVCMOS33")),
+
+        Resource("button_reset",  0, PinsN("C12", dir="i"),
+                 Attrs(IOSTANDARD="LVCMOS33")),
+        Resource("button_center", 0, Pins("N17",  dir="i"),
+                 Attrs(IOSTANDARD="LVCMOS33")),
+        Resource("button_up",     0, Pins("M18",  dir="i"),
+                 Attrs(IOSTANDARD="LVCMOS33")),
+        Resource("button_left",   0, Pins("P17",  dir="i"),
+                 Attrs(IOSTANDARD="LVCMOS33")),
+        Resource("button_right",  0, Pins("M17",  dir="i"),
+                 Attrs(IOSTANDARD="LVCMOS33")),
+        Resource("button_down",   0, Pins("P18",  dir="i"),
+                 Attrs(IOSTANDARD="LVCMOS33")),
+
+        Resource("vga", 0,
+                 Subsignal("r",  Pins("A3 B4 C5 A4", dir="o")),
+                 Subsignal("g",  Pins("C6 A5 B6 A6", dir="o")),
+                 Subsignal("b",  Pins("B7 C7 D7 D8", dir="o")),
+                 Subsignal("hs", Pins("B11", dir="o")),
+                 Subsignal("vs", Pins("B12", dir="o")),
+                 Attrs(IOSTANDARD="LVCMOS33")),
+
+        *SDCardResources(0,
+                         clk="B1",  cmd="C1",  cd="A1",
+                         dat0="C2", dat1="E1", dat2="F1", dat3="D2",
+                         attrs=Attrs(IOSTANDARD="LVCMOS33")),
+        Resource("sd_card_reset", 0, Pins("E2", dir="o"),
+                 Attrs(IOSTANDARD="LVCMOS33")),
+
+        Resource("accelerometer", 0,                        # ADXL362
+                 SPIResource("spi", cs="D15", clk="F15", mosi="F14", miso="E15",
+                             attrs=Attrs(IOSTANDARD="LVCMOS33")),
+                 Subsignal("int1", Pins("B13", dir="i")),
+                 Subsignal("int2", Pins("C16", dir="i")),
```

Comment:
I think this would be nicer as a single 2-bit `int` subsignal, given that logically it's a set of interrupts.